app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes, class_name: 'Like'
+  has_many :comments, class_name: 'Comment'
+
+  after_save :update_post_counter
```

Comment:
✅ Good job using a callback to update the posts counter every time a new post has been saved to database 👍🏻